tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +83,16 @@ func (tt *tester) runLoop() {
 			plog.Printf("[round#%d case#%d] recovered failure", i, j)
 
 			if !tt.consistencyCheck {
+				plog.Printf("[round#%d case#%d] canceling the stressers...", i, j)
```

Comment:
also we probably should remove this 5 second dial timeout:

https://github.com/coreos/etcd/blob/master/tools/functional-tester/etcd-tester/stresser.go#L64.

gRPC will do internal redial for us. this limits the total dial timeout. the redial might fail. we do not want it fail.